src/main/java/cn/nukkit/block/BlockLever.java

```diff
@@ -67,10 +67,8 @@ public boolean onActivate(Item item, Player player) {
         this.level.getServer().getPluginManager().callEvent(new BlockRedstoneEvent(this, isPowerOn() ? 15 : 0, isPowerOn() ? 0 : 15));
         this.setDamage(this.getDamage() ^ 0x08);
 
-        boolean redstone = this.level.getServer().isRedstoneEnabled();
```

Comment:
The redstone variable was used later, so removing it caused a build issue

Reply:
Yes I see it, fixed at 929fd26f836f1d7cdb69c9fbe33310c3a9717b2b

Was a merge conflict issue.

Reply:
Another fix at 6996f4a183a0ec5fa518dfddf997df43ca96952c